scripts/i.spectral/i.spectral.html

```diff
@@ -30,21 +30,26 @@ <h2>EXAMPLE</h2>
 <div class="code"><pre>
 g.region raster=lsat7_2002_10 -p
 LIST=`g.list type=raster pattern="lsat7_2002_[1-5,7]0" separator=","`
+d.mon start=cairo output=plot.png
 i.spectral input=$LIST coordinates=637502.25,221744.25
```

Comment:
```suggestion
i.spectral raster=$LIST coordinates=637502.25,221744.25
```

Reply:
(TODO: this also needs to be fixed above in line 17 which I cannot reach from the web interface)

Reply:
Thanks for the notice, I need to be more consistent.